### PR TITLE
add carousel

### PR DESCRIPTION
**Pull request by Anastasiya-Shir**

### CODE REVIEW

File: Carousel/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+function showImage(i) {
+  counter += i;
+  const images = document.getElementsByClassName("picture");
+  const dots = document.getElementsByClassName("dot");
+  console.log(dots)
+  for (i = 0; i < images.length; i++) {
+    images[i].style.display = "none";
+  }
```
Comment (kathrynowy) [inline Carousel/index.js]: remove clg, separate js

File: Carousel/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  for (i = 0; i < dots.length; i++) {
+    console.log(i);
+    console.log(dots[i]);
```
Comment (kathrynowy) [inline Carousel/index.js]: remove

File: Carousel/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    dots[i].className = dots[i].className.replace("active", " ");
+  }
+  if (counter > images.length - 1) {
+    counter = 0;
+  }
+  if (counter < 0) {
+    counter = images.length - 1;
+  }
```
Comment (kathrynowy) [inline Carousel/index.js]:
```suggestion
  if (counter > images.length - 1) {
    counter = 0;
  }
  
  if (counter < 0) {
    counter = images.length - 1;
  }
```

File: Carousel/styles.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  position: relative;
+}
+
```
Comment (kathrynowy) [inline Carousel/styles.css:12]: remove an empty line

File: Carousel/styles.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+img {
+  width: 50%;
+  height: 80vh;
+
```
Comment (kathrynowy) [inline Carousel/styles.css]: ?

File: Carousel/styles.css [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+@media(max-width: 949px) {
+  img {
+    height: 50%;
+  }
+}
+
+
+
+.fa-solid {
+  background-color: #808080;
```
Comment (kathrynowy) [inline Carousel/styles.css:30]: please fix the indentation

File: Carousel/styles.css [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+.dots {
+  display: flex;
+  flex-direction: row;
+  margin: 20px;
+}
+
+.active {
+  background-color: #000;
+
+}
```
Comment (kathrynowy) [inline Carousel/styles.css]: eof

File: shoping list/fonts/OFL.txt [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+efforts of academic and linguistic communities, and to provide a free and
+open framework in which fonts may be shared and improved in partnership
+with others.
+
+The OFL allows the licensed fonts to be used, studied, modified and
+redistributed freely as long as they are not sold by themselves. The
+fonts, including any derivative works, can be bundled, embedded, 
+redistributed and/or sold with any software provided that any reserved
+names are not used by derivative works. The fonts and derivatives,
+however, cannot be released under any other type of license. The
```
Comment (kathrynowy) [inline shoping list/fonts/OFL.txt]: do you need this file?

File: shoping list/idex.html [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Shoping list</title>
+  <link rel="stylesheet" href="./index.css">
+  <link rel="shortcut icon" href="./img/icon.jpg" type="image/x-icon">
+</head>
+<body>
+
+  <div class="container">
+
```
Comment (kathrynowy) [inline shoping list/idex.html]: fix html

File: shoping list/index.css [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  li {
+    font-size: 20px;
+  }
+
+  input {
+    margin: 2;
+  }
+}
```
Comment (kathrynowy) [inline shoping list/index.css]: eof

File: shoping list/index.js [hunk 11]
```diff
@@ -0,0 +1,27 @@
+let list = document.querySelector(".list");
+console.log(list);
+let input = document.querySelector("input");
+console.log(input);
+let pen = document.querySelector("img");
+console.log(pen);
```
Comment (kathrynowy) [inline shoping list/index.js]: remove clg and fix js indentation

File: shoping list/index.js [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+pen.addEventListener('click', () => {
+  list.remove();
+});
+
+input.addEventListener("keydown", (event) => {
+
```
Comment (kathrynowy) [inline shoping list/index.js]: remove an empty line

File: watch/fonts/OFL.txt [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+development of collaborative font projects, to support the font creation
+efforts of academic and linguistic communities, and to provide a free and
+open framework in which fonts may be shared and improved in partnership
+with others.
+
+The OFL allows the licensed fonts to be used, studied, modified and
+redistributed freely as long as they are not sold by themselves. The
+fonts, including any derivative works, can be bundled, embedded, 
+redistributed and/or sold with any software provided that any reserved
+names are not used by derivative works. The fonts and derivatives,
```
Comment (kathrynowy) [inline watch/fonts/OFL.txt]: remove this file

File: watch/index.css [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+
+@media(max-width:296px) {
+  .container-watch {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: 10px;
+  }
+}
```
Comment (kathrynowy) [inline watch/index.css]: eof

File: watch/index.js [hunk 15]
```diff
@@ -0,0 +1,42 @@
+let date = new Date();
+const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
```
Comment (kathrynowy) [inline watch/index.js]: constant variables (when it's a static constant value - we name with capital letters)

DAY_NAMES

File: watch/index.js [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+
+  if (minutes > 9) {
+    minuteClass.innerHTML = minutes;
+  } else {
+    minuteClass.innerHTML = `0${minutes}`;
+  }
+
+  date = new Date();
+  setT = setTimeout(getDate, 2000, date.getDay(), date.getHours(), date.getMinutes());
+
```
Comment (kathrynowy) [inline watch/index.js]: fix js indentation

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: watch/index.css (hunk 14 above)
Comment (Anastasiya-Shir) [reply watch/index.css]: у меня css почему-то удаляет пустую строку

File: watch/index.js [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  if (minutes > 9) {
+    minuteClass.innerHTML = minutes;
+  } else {
+    minuteClass.innerHTML = `0${minutes}`;
+  }
+
+  date = new Date();
+  setT = setTimeout(getDate, 2000, date.getDay(), date.getHours(), date.getMinutes());
```
Comment (kathrynowy) [inline watch/index.js]: what for do you pass date.getDay(), date.getHours(), date.getMinutes() this?
you can get this just inside the function

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: watch/index.js (hunk 17 above)
Comment (Anastasiya-Shir) [reply watch/index.js]: removed, now i don't know why i did it

File: Carousel/index.html [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Document</title>
+  <script src="https://kit.fontawesome.com/4ca0456cfc.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+  <div class="container">
+
```
Comment (kathrynowy) [inline Carousel/index.html]: we don't add an empty line after ONLY opening a tag and before closing it

File: Carousel/index.html [hunk 19]
```diff
@@ -0,0 +1,39 @@
...
+    <div class="dots">
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+      <div class="dot"></div>
+    </div>
+
```
Comment (kathrynowy) [inline Carousel/index.html]: as i described before

we don't add an empty line before closing an element

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: Carousel/index.html (hunk 18 above)
Comment (Anastasiya-Shir) [reply Carousel/index.html]: fixed

File: Carousel/index.html (hunk 19 above)
Comment (Anastasiya-Shir) [reply Carousel/index.html]: fixed

File: Carousel/styles.css (hunk 6 above)
Comment (Anastasiya-Shir) [reply Carousel/styles.css:30]: fixed

File: shoping list/idex.html (hunk 9 above)
Comment (Anastasiya-Shir) [reply shoping list/idex.html]: fixed

File: watch/index.js (hunk 16 above)
Comment (Anastasiya-Shir) [reply watch/index.js]: fixed

**Pull request merged**